processor/spanmetricsprocessor/processor_test.go

```diff
@@ -376,6 +378,37 @@ func TestProcessorConsumeTraces(t *testing.T) {
 			},
 			traces: []pdata.Traces{spanWithLargeTimestamp},
 		},
+		{
+			name:                   "Test that metric renaming works",
```

Comment:
Should we have another case to cover the default metric names `calls_total` and `latency`?